com.beeper.Beeper.yaml

```diff
@@ -0,0 +1,89 @@
+  - --require-version=1.7.0
+  # Required due to being a GUI application
+  - --socket=x11
+  # Required to make sure x11 performance is achived on all platforms
+  # At least that's what the legends tell. it might be worth experimenting
+  # with dropping this permission.
+  - --share=ipc
+  # Required to provide Call functionality
+  - --socket=pulseaudio
+  - --device=all
+  # As a chat application, networking is required
+  - --share=network
+  # Required for appindicator in various desktop environments
+  - --talk-name=org.kde.StatusNotifierWatcher
+  # Required to allow screensaver/idle inhibition such as during video calls
+  - --talk-name=org.freedesktop.ScreenSaver
+  # Required for advanced input methods e.g. writing CJK languages
+  - --talk-name=org.freedesktop.portal.Fcitx
+  # Required to store access tokens (persistent logins)
+  - --filesystem=xdg-run/keyring
+  # Required for experimental wayland support
+  - --filesystem=xdg-run/pipewire-0
+  # For correct cursor scaling under Wayland
+  - --env=XCURSOR_PATH=/run/host/user-share/icons:/run/host/share/icons
+cleanup:
+  - /include
+  - /lib/pkgconfig
+  - /lib/*.la
+  - /lib/*.a
+  - /share/pkgconfig
+  - /share/aclocal
+  - /share/gtk-doc
+  - /share/doc
+  - /share/info
+  - /share/man
+  - /man
+modules:
+  - shared-modules/libsecret/libsecret.json
+  - name: beeper
+    buildsystem: simple
+    build-commands:
+      - chmod +x Beeper.AppImage
+      - ./Beeper.AppImage --appimage-extract
+
+      - desktop-file-edit --set-icon ${FLATPAK_ID} --set-key Exec --set-value 'startbeeper
+        %u' squashfs-root/beeper.desktop
+      - install -D squashfs-root/beeper.desktop /app/share/applications/${FLATPAK_ID}.desktop
+      - install -D startbeeper /app/bin/
+      - mkdir -p /app/share/icons/hicolor
+      - cp -r squashfs-root/usr/share/icons/hicolor/* ${FLATPAK_DEST}/share/icons/hicolor
+      # flatpak doesn't support icons larger than 512x512
+      - rm -rf ${FLATPAK_DEST}/share/icons/hicolor/1024x1024
+      - mv squashfs-root /app/bin/beeper
+      - rm -rf squashfs-root/resources/icons/win
+      - rm Beeper.AppImage
+    sources:
+      - type: script
+        dest-filename: startbeeper
+        commands:
+          - for i in {0..9}; do
+          - test -S $XDG_RUNTIME_DIR/discord-ipc-$i || ln -sf {app/com.discordapp.Discord,$XDG_RUNTIME_DIR}/discord-ipc-$i;
+          - done
+          - export FLATPAK_ID="${FLATPAK_ID:-com.beeper.Beeper}"
+          - export TMPDIR="$XDG_RUNTIME_DIR/app/$FLATPAK_ID"
+          - zypak-wrapper /app/bin/beeper/beeper "$@"
+
+      - type: file
+        only-arches: [x86_64]
+        url:  https://download.beeper.com/versions/3.77.21/linux/appImage/x64
+        sha512: cd070a6561714db5c12a1306a1e222c7e215aaadc7fbf40050fdc7b71bf5313a1eb704f0bb0efecd9200fec1c2c96e010a5503f8a71d54eb90b39385ffc0617d
+        dest_filename: Beeper.AppImage
+      - type: file
+        path: com.beeper.Beeper.metainfo.xml
+        x-checker-data:
```

Comment:
This is in the wrong place.

Reply:
to the proper source.

Reply:
but then you did it. https://github.com/flathub/flathub/pull/4548/commits/4ab80e93327e6b56b7c486aabf6fc413c7328d24